source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,117 @@
+.. _glossary:
+
+=========
+Glossary
+=========
+
+Machine learning terms
+======================
+
+.. glossary::
+    :sorted:
+
+    Attribute
+        A particular property or quality of a real object or an event.
+        Has a defined type and domain.
+
+    Classification
+        A supervised machine-learning problem to assign categorical :term:`labels<Label>`
+        to :term:`feature vectors<Feature vector>` describing input objects.
+
+    Dataset
+        A collection of :term:`instances<Instance>`.
+
+    Feature
+        A pair of :term:`attribute` information and a corresponding value.
+
+    Feature vector
+        A vector that encodes information about single real object or an event.
+        Contains at least one :term:`feature`.
+
+    Inference
+        A process of obtaining outputs from the :term:`model` created on the :term:`training` stage
+
+    Instance
+        A vector of at least one :term:`feature` value and zero or more :term:`label<Label>` values.
+        Each of :term:`labels<Label>` and :term:`features<Feature>` can encode different objects or events.
+
+    Label
+        :term:`Feature` of some object or event which is present on the :term:`training` stage but
+        unavaliable on the :term:`inference` stage.
```

Comment:
* it's not only unavailable, but it's something you predict at the inference stage -- I think it should be mentioned
* **at** the inference stage

Reply:
Reformulated.